hs_core/templates/search/search.html

```diff
@@ -253,50 +379,122 @@ <h4 class="panel-title">
+                }
+            }
+            markers = [];
+        }
+
+        var checkDuplicatePointResults = function (modified_points_data, test){
+            var test_lat = parseFloat(test.north);
+            var test_lng = parseFloat(test.east);
+            var existed = false;
+            modified_points_data.forEach(function(item){
+                var item_lat = parseFloat(item.north);
+                var item_lng = parseFloat(item.east);
+
+                var lat_diff = Math.abs(test_lat - item_lat);
+                var lng_diff = Math.abs(test_lng - item_lng);
+                if (lat_diff < minFloatingNumber && lng_diff < minFloatingNumber) {
+                    item.info_link = item.info_link + '<br />' + '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                    item.counter++;
+                    existed = true;
+                }
             });
+            if (!existed){
+                test.info_link = '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                test.counter = 1;
+                modified_points_data.push(test);
+            }
+        }
+
+        var checkDuplicateBoxResults = function (modified_boxes_data, test){
+            var test_lat = (parseFloat(test.northlimit) + parseFloat(test.southlimit)) / 2;
+            var test_lng = (parseFloat(test.eastlimit) + parseFloat(test.westlimit)) / 2;
+            var existed = false;
+            modified_boxes_data.forEach(function(item){
+                var item_lat = (parseFloat(item.northlimit) + parseFloat(item.southlimit)) / 2;
+                var item_lng = (parseFloat(item.eastlimit) + parseFloat(item.westlimit)) / 2;
+
+                var lat_diff = Math.abs(test_lat - item_lat);
+                var lng_diff = Math.abs(test_lng - item_lng);
+
+                if (lat_diff < minFloatingNumber && lng_diff < minFloatingNumber) {
+                    item.info_link = item.info_link + '<br />' + '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                    item.counter++;
+                    existed = true;
+                }
+            });
+            if (!existed){
+                test.info_link = '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                test.counter = 1;
+                modified_boxes_data.push(test);
+            }
         }
 
-        var createPointResourceMarker = function (lat, lng, info_content, bounds, info_window, rectangle, markers) {
+        var createPointResourceMarker = function (lat, lng, info_content, counter, info_window, rectangle) {
             var latlng = new google.maps.LatLng(lat, lng);
             var marker = new google.maps.Marker({
                 map: map,
-                icon: '//maps.google.com/intl/en_us/mapfiles/ms/micons/red-dot.png',
+                icon: '//cdn.rawgit.com/Concept211/Google-Maps-Markers/master/images/marker_red' + counter + '.png',
```

Comment:
@hyi If we download all markers, we would have to store around 200 markers' images. I am thinking to customize the markers by SVG. Instead of getting images from cdn sever, we can only update the number in the SVG graph. This might be a better way to solve this.